### PR TITLE
Service History Page

### PR DESCRIPTION
**Pull request by harshida97**
New table and Cards are created and updated in Service History page

### CODE REVIEW

File: src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx
Comment (JaseelaAlimahamed) [inline src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx:1]: make this component inside components/adminComponents/CategorySubCategory/AddEditCategory/AddCategory.jsx please

File: src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        >
+          <svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M11 4.00023H4C3.46957 4.00023 2.96086 4.21094 2.58579 4.58601C2.21071 4.96109 2 5.46979 2 6.00023V20.0002C2 20.5307 2.21071 21.0394 2.58579 21.4144C2.96086 21.7895 3.46957 22.0002 4 22.0002H18C18.5304 22.0002 19.0391 21.7895 19.4142 21.4144C19.7893 21.0394 20 20.5307 20 20.0002V13.0002M18.5 2.50023C18.8978 2.1024 19.4374 1.87891 20 1.87891C20.5626 1.87891 21.1022 2.1024 21.5 2.50023C21.8978 2.89805 22.1213 3.43762 22.1213 4.00023C22.1213 4.56284 21.8978 5.1024 21.5 5.50023L12 15.0002L8 16.0002L9 12.0002L18.5 2.50023Z"
```
Comment (JaseelaAlimahamed) [inline src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx:59]: save svg files in assets and call it here

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]

File: src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx
Comment (harshida97) [reply src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx:1]: update folder structure

File: src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx (hunk 1 above)
Comment (harshida97) [reply src/components/CategorySubCategory/AddEditCategory/AddCategory.jsx:59]: this issue corrected. svg files saved in assets and call it here.

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]
proper folder structure

File: src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx.jsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+          src={image|| "https://via.placeholder.com/250x150"}
+          alt="Category"
+          className="w-64 h-64 object-cover rounded-lg mt-6"
+        />
+        <button
+          onClick={handleEditImage}
+          className="ml-0 p-2  bg-transparent border-none cursor-pointer text-gray-600"
+        >
+          <svg
+            width="24"
```
Comment (JaseelaAlimahamed) [inline src/components/adminComponents/CategorySubCategory/AddEditSubCategory/EditSubCategory.jsx.jsx:74]: svg file in code?

**Comment by harshida97** [comment]
Maam ith njan cheytha page alla

**Pull request merged**